ipatests/test_cmdline/test_help.py

```diff
@@ -174,3 +174,18 @@ def test_multiline_description():
         api.Backend.cli.run(['trust-add', '-h'])
 
     assert unicode(api.Command.trust_add.doc).strip() in help_ctx.stdout
+
+
+def test_bz1428690():
+    """
+    Test for BZ#1428690 - ipa-backup does not create log file at /var/log/
+    :return: None
```

Comment:
Please add a self-explanatory comment, for instance:
When ipa-backup is executed on a node where IPA is not configured, the error message should not refer to logs at /var/log/ because the log file has not even been created. The command should print that IPA is not configured.